PopupBridge/src/main/java/com/braintreepayments/api/PopupBridgeClient.kt

```diff
@@ -21,6 +22,7 @@ class PopupBridgeClient @SuppressLint("SetJavaScriptEnabled") internal construct
     private var navigationListener: PopupBridgeNavigationListener? = null
     private var messageListener: PopupBridgeMessageListener? = null
     private var errorListener: PopupBridgeErrorListener? = null
+    private var venmoInstalled: Boolean = false
```

Comment:
Nit - we should be able to make this a val here since it's set in the init.

```suggestion
    private val venmoInstalled: Boolean = false
```